server/controllers/user.js

```diff
@@ -21,3 +21,26 @@ exports.searchUsers = asyncHandler(async (req, res, next) => {
 
   res.status(200).json({ users: users });
 });
+
+// @route GET /users/isSitter/
+// @desc GET allowed parameters within User model
+// @access Private
+exports.getIsSitter = asyncHandler(async (req, res, next) => {
+  const id = req.user.id;
+
+  const user = await User.findById(id);
+  const { isSitter } = user;
+
+  res.status(200).json({ isSitter });
+});
+
+// @route PATCH /users/isSitter/
+// @desc Update allowed parameters within User model
+// @access Private
+exports.setIsSitter = asyncHandler(async (req, res, next) => {
+  const id = req.user.id;
+  const isSitter = req.body.isSitter;
```

Comment:
Need to validate request body

Reply:
Thanks, I made those changes as requested. Let me know if there's any issue with the validation.